source/install/prod-ubuntu-16.04.rst

```diff
@@ -0,0 +1,489 @@
+..  _prod-ubuntu-1604:
+
+Production Install on Ubuntu 16.04 LTS
+======================================
+
+Install Mattermost in production mode on one, two or three machines, using the following steps:
+
+- `Install Ubuntu Server (x64) 16.04 LTS`_
+- `Set up Database Server`_
+- `Set up Mattermost Server`_
+- `Set up NGINX Server`_
+- `Post-Installation Mattermost Configuration`_
+
+
+Install Ubuntu Server (x64) 16.04 LTS
+-------------------------------------
+
+1. Set up 3 machines with Ubuntu 16.04 with 2GB of RAM or more. The
+   servers will be used for the Proxy, Mattermost (must be
+   x64), and Database.
+
+   -  **Optional:** You can also use a **1 machine setup** (Proxy, Mattermost and Database on one machine) or a **2 machine setup** (Proxy and Mattermost on one machine, Database on another) depending on your data center standards.
+
+2. Make sure the system is up to date with the most recent security
+   patches.
+
+   ::
+
+       sudo apt update
+       sudo apt upgrade
+
+
+Set up Database Server
```

Comment:
Thanks @enahum, I think we can take this PR as is and add MySQL in a new PR if you're interested @okin . Agree the end goal would be to have Postgres and MySQL in both install guides. PR for adding MySQL to 14.04 guide was just merged. https://github.com/mattermost/docs/pull/573